### PR TITLE
feat(EMS-2139): No PDF - Your business - Nature of business

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction ✏️ 
This PR removes `totalEmployeesInternational` field and renamed hints and labels on nature of your business page

## Resolution ✔️ 
* Removed `totalEmployeesInternational`
* Renamed label for `employeesUK` and removed fieldset
* Changed label and hint for `goodsOrServicesSupplied`
* Updated tests

### CODE REVIEW

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/nature-of-business-page.spec.js [hunk 1]
```diff
@@ -97,31 +96,11 @@ context('Insurance - Your business - Nature of your business page - As an Export
       cy.checkText(field.suffix(), FIELDS.NATURE_OF_YOUR_BUSINESS[fieldId].SUFFIX);
     });
 
-    it('should display save and go back button', () => {
-      cy.checkText(saveAndBackButton(), BUTTONS.SAVE_AND_BACK);
-    });
```
Comment (ttbarnes) [inline e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/nature-of-business-page.spec.js:102]: I imagine the save and back button/functionality should remain?

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/validation/nature-of-business-employees-validation.spec.js [hunk 2]
```diff
@@ -22,6 +21,8 @@ const {
 
 const baseUrl = Cypress.config('baseUrl');
 
+const expectedErrorsCorrectValidation = 2;
```
Comment (ttbarnes) [inline e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/validation/nature-of-business-employees-validation.spec.js:24]: This variable name is confusing, I don't think we should associate "correct" with "validation".

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/nature-of-business-page.spec.js (hunk 1 above)
Comment (Zainzzkk) [reply e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/nature-of-business-page.spec.js:102]: its part of the next test under this.  Was duplicated

File: e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/validation/nature-of-business-employees-validation.spec.js (hunk 2 above)
Comment (Zainzzkk) [reply e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/validation/nature-of-business-employees-validation.spec.js]: reverted to just number

File: src/ui/server/controllers/insurance/business/save-data/business/index.test.ts [hunk 3]
```diff
@@ -31,7 +30,7 @@ describe('controllers/insurance/business/save-data/business', () => {
   describe('when errorList is provided', () => {
     const mockValidationErrors = generateValidationErrors(EMPLOYEES_UK, 'error', {});
 
-    it(`should call api.keystone.application.update.business with ${GOODS_OR_SERVICES}, ${YEARS_EXPORTING}, ${EMPLOYEES_INTERNATIONAL} but not ${EMPLOYEES_UK}`, async () => {
+    it(`should call api.keystone.application.update.business with ${GOODS_OR_SERVICES} and ${YEARS_EXPORTING} but not ${EMPLOYEES_UK}`, async () => {
```
[the remark this answers is not in the document]
Comment (Zainzzkk) [reply src/ui/server/controllers/insurance/business/save-data/business/index.test.ts:33]: this is for save and back with validation errors

**Pull request merged**